curriculum/challenges/english/10-coding-interview-prep/take-home-projects/chart-the-stock-market.english.md

```diff
@@ -9,14 +9,14 @@ forumTopicId: 302363
 
 ## Description
 <section id='description'>
-<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='http://watchstocks.herokuapp.com/' target='_blank'>http://watchstocks.herokuapp.com/</a> and deploy it to Heroku.
+<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='http://watchstocks.herokuapp.com/' target='_blank'>http://watchstocks.herokuapp.com/</a>.
```

Comment:
We should always be linking to the `https` version of a site.  The herokuapp.com `http` links do not automatically forward to the `https` version, so we should specify the `https` version. 
```suggestion
<strong>Objective:</strong> Build a full stack JavaScript app that is functionally similar to this: <a href='https://watchstocks.herokuapp.com/' target='_blank'>https://watchstocks.herokuapp.com/</a>.
```